website/src/contracts.tsx

```diff
@@ -17,19 +17,17 @@ export function ProtocolContractsTable({ networkId }: { networkId: number }) {
     <Table>
       <tbody>
         <tr>
-          <th>{'Contract'}</th>
-          <th>{'Address'}</th>
+          <th>Contract</th>
+          <th>Address</th>
```

Comment:
These should probably be translated, adding myself a TODO for that.

Reply:
want to rollback?

Reply:
No no, as it is now, it makes no sense to use interpolation. This is a good change. :P